nbs/tsdataset.ipynb

```diff
@@ -54,13 +54,15 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #102.                ids = ids + [uid]
</pre><p>Please use <code>ids.append(uid)</code> here, this copies the whole list every time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>